tableauserverclient/server/endpoint/workbooks_endpoint.py

```diff
@@ -199,7 +199,14 @@ def _get_wb_preview_image(self, workbook_item):
 
     # Publishes workbook. Chunking method if file over 64MB
     @api(version="2.0")
-    def publish(self, workbook_item, file_path, mode, connection_credentials=None):
+    @parameter_added_in(connections='2.8')
+    def publish(self, workbook_item, file_path, mode, connection_credentials=None, connections=None):
+
+        if connection_credentials is not None:
+            import warnings
+            warnings.warn("connection_credendials is being deprecated. Use connections instead, see http://...",
```

Comment:
Typo in connection_credendials. Also I'm not too familiar with connection_credentials but should this warning also be in datasources_endpoint.py?

Reply:
If this were supported in Datasources then yes, but it turns out my code is ahead of the curve here, datasources don’t do multi-credential publishing yet.